relay/cmd/flags/flags.go

```diff
@@ -202,17 +202,11 @@ var (
 		EnvVar:   common.PrefixEnvVar(envVarPrefix, "MAX_CONCURRENT_GET_CHUNK_OPS_CLIENT"),
 		Value:    1,
 	}
-	BlsOperatorStateRetrieverAddrFlag = cli.StringFlag{
-		Name:     common.PrefixFlag(FlagPrefix, "bls-operator-state-retriever-addr"),
-		Usage:    "Address of the BLS operator state retriever",
-		Required: true,
-		EnvVar:   common.PrefixEnvVar(envVarPrefix, "BLS_OPERATOR_STATE_RETRIEVER_ADDR"),
-	}
-	EigenDAServiceManagerAddrFlag = cli.StringFlag{
-		Name:     common.PrefixFlag(FlagPrefix, "eigen-da-service-manager-addr"),
-		Usage:    "Address of the Eigen DA service manager",
-		Required: true,
-		EnvVar:   common.PrefixEnvVar(envVarPrefix, "EIGEN_DA_SERVICE_MANAGER_ADDR"),
+	AddressDirectoryFlag = cli.StringFlag{
```

Comment:
here is another case where the term "address directory" is used, instead of "EigenDA Directory"